### ISSUE TITLE
Onboarding: Damilare Ogunsanya

### ISSUE THREAD

**Issue by DMLROG** [hackforla/guides#102]
### Overview
We need an unbiased review of the gather presentation so that we can improve it.

#### Additional Details
You will be working on guide during this issue, but the purpose of this issue is to evaluate and improve the Gather Examples and Interview Teams presentation.  
- Any notes you have about the guide you will be making should be noted in the issue related to the Guide you choose.
- Any notes about the Gather Examples and Interview Teams presentation should be in the document you create.

### Action Items
- [ ] Create a Google doc, with the name ```GT: Notes on Gather presentation - YOUR NAME HERE``` in the Gather Examples folder (customize with your name)
   - [ ] Add the link to the document and its name in the Resources section below
- [ ] Add to the notes document as you go, with observations about the Gather Presentation.  You will encounter some issues, such as font size or other types of formatting changes, what we are looking for though is material changes to the instructions.  For instance, if something could use a screenshot to make it clearer, and it doesn't have one or when you try to follow the steps, something is missing.
   - [ ] If while looking at the presentation (not the guide), you encounter any words that you think should be added to our glossary,  please add them to the Glossary issue as comments.
- [ ] Review the Presentation for Gathering Examples
- [ ] Pick a guide 
   - [ ] Note the Name of guide and URL of the guide in the comments below
- [ ] Use the presentation to work on the guide
   - [ ] Leaving comments in the presentation if you have a suggested improvement
   - [ ] Putting any notes about the guide either in the guide or in the issue for the guide (as appropriate)
- [ ] Clean up your presentation notes
- [ ] Present to the Guides Team

### Resources/Instructions
- #88 
- [Guides Tracker](https://github.com/orgs/hackforla/projects/3)
- [Gather Examples and Interview Teams presentation](https://docs.google.com/presentation/d/1gkPpgnOPv7lBckoLic3HAA7z_IkeTnCw9ImQp9hkujM/edit)
- [PM Templates and Guides Project board](https://github.com/hackforla/product-management/projects/2)
- Folder for Team notes and Guide Presentation: [Gather Examples](https://drive.google.com/drive/u/0/folders/15ybydhWpIYr9a3MyE2FZf1g-SJs-23Jl)
- Your presentation notes document file: 
- Guide you are working on
   - Name of guide
   - URL of guide Issue

### When you do your update please provide
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by DMLROG** [comment]
Progress: - I have skimmed through the GP and it's getting clear to me what needs to be done as per creating guides. I plan to still go through each slide thoroughly and that's when I'll document my notes on what could be done better.
- I have picked a guide to work on and I've started looking at the issue and previous correspondence on it (a number of people have worked on this to a point earlier) 
Blockers: The main blocker at the moment, is taking time to study and understand what already has been done previously on the guide I have picked to work on. It's a little overwhelming. [Create a Guide: Creating Eco-system mapping for topic area of project #80](https://github.com/hackforla/product-management/issues/80)
Availability: 6 hours a week seems fair.
ETA: Honestly? Anything quicker than a month or two seems unrealistic right now. But let's see how I progress.
Pictures (if necessary): None at the moment.

**Comment by SheIsBukki** [comment]
@DMLROG Thanks for the update. We also looked at your observation and it is now resolved

**Comment by DMLROG** [comment]
Hi @SheIsBukki I'm not clear on what exactly you've stated as "now resolved". Care to explain a bit more? Thanks.

**Comment by ExperimentsInHonesty** [comment]
@DMLROG We responded to your observation in the observation document that you created.  It was similar to other team members' observation about the index, so we have addressed it and marked it as addressed, so we don't go back over it.

**Comment by ExperimentsInHonesty** [comment]
@DMLROG 
Please provide update.  
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by ExperimentsInHonesty** [comment]
@DMLROG 
Please provide update.  
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

Next message: 2022-04-11

**Comment by DMLROG** [comment]
Progress: As per the Gather Examples Presentation, I have been able to look through the GP (up until Slide 28) while working on the Guide simultaneously too. I have finally added my comments and recommendations on making the GP document better as promised here - [GT: Notes on Gather presentation - Damilare Ogunsanya](https://docs.google.com/document/d/1i025lcjpGCU_CaWLQrGrcK3TZLhVEJFGpjD1oNAGHNc/edit?usp=sharing)

I have made some progress working on the guide too and you can find an update under that issue.
[Create a Guide: Creating Eco-system mapping for topic area of project #80](https://github.com/hackforla/product-management/issues/80)

Blockers: None for GP. But for the guide, I don't have access to a couple of files previously worked on (I have provided their links under the issue). I also have a few questions that their answers will provide me with much-needed guidance.
 
Availability: 6 hours a week

ETA: Still can't say yet.

Pictures (if necessary): One screenshot is attached in the guide issue.

**Comment by ExperimentsInHonesty** [comment]
We got through the first 3 notes in your doc.  We will review the rest at our next meeting.  Thank you for the observations!

**Comment by ExperimentsInHonesty** [comment]
@DMLROG we have reviewed your comments and put notes in the doc.

**Comment by DMLROG** [comment]
@ExperimentsInHonesty I've seen the comments in the doc and they've been noted. I'll update with more relevant observations if I find any.

Could you please look through the comments on my guide issue? Getting feedback on the questions I asked is actually a major blocker at the moment.

Here— https://github.com/hackforla/product-management/issues/80

**Comment by ExperimentsInHonesty** [comment]
@DMLROG see my comment here about access https://github.com/hackforla/product-management/issues/80#issuecomment-1095167598

We need to figure what kind of guidance we need to give to people working on these guides, for when they can't get access.  S for instance, with this, you could have reached out to your counterpart on the Food Oasis and asked them for access.  Where do you think that should go in the slides?

**Comment by DMLROG** [comment]
@ExperimentsInHonesty Comment noted.
Regarding reaching out to counterparts for access on the GP slides- I think Slide 28 is a good place to add that. 
Just under the text as I've shown in the attached screenshot.

We can say **"Reaching out to counterpart is also useful when you need to be granted access to the Project Drive to view resources relevant to your guide"** 

Also left a new comment in the guide issue [here](https://github.com/hackforla/product-management/issues/80#issuecomment-1097942623)



![Screenshot 2022-04-13 122650](https://user-images.githubusercontent.com/101733238/163170805-67c7fa8e-03dd-4ebe-98a1-ba8edace0775.png)

**Comment by ExperimentsInHonesty** [comment]
@DMLROG We added [this slide](https://docs.google.com/presentation/d/1gkPpgnOPv7lBckoLic3HAA7z_IkeTnCw9ImQp9hkujM/edit#slide=id.g121e6985d41_1_46).  Can you review and see if this helps.  If there is any other information to add, please let us know.

**Comment by DMLROG** [comment]
@ExperimentsInHonesty Yes. I think that works fine and explains what needs to be done when someone needs access.
No further information to add at the moment.

**Comment by ExperimentsInHonesty** [comment]
@DMLROG Responded to your message on the issue https://github.com/hackforla/product-management/issues/80#issuecomment-1104090278

**Comment by SheIsBukki** [comment]
@DMLROG Bonnie replied to the issues you raised in the guide issue https://github.com/hackforla/product-management/issues/80#issuecomment-1104090278

**Comment by SheIsBukki** [comment]
@DMLROG Please provide an update

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by ExperimentsInHonesty** [comment]
@DMLROG 
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

if no response On 5/4 send him two week message

**Comment by DMLROG** [comment]
@ExperimentsInHonesty @SheIsBukki 

Progress: At the moment, I have no further comments about the Guides Gathering Examples Presentation.
For the guide I'm working on, I'm looking through all Hack for LA projects to find relevant resources related to my issue (finished looking through 2 projects). 
And, I'm also still going through previously gathered assets so I can summarise what has already been done and give direction.
My most recent comment on the issue can be found [here](https://github.com/hackforla/product-management/issues/80#issuecomment-1117329518)
Blockers: None. All blockers in my last comment have been cleared.
Availability: Not so much this week. Maybe 2-3 hours.
ETA: Still can't say, unfortunately.
Pictures (if necessary): None, at the moment.

**Comment by ExperimentsInHonesty** [comment]
send him 3 week message on 2022-05-25

**Comment by DMLROG** [comment]
@SheIsBukki Yes, I had already started gathering examples (and I still am currently) when you asked, as shown in my previous comments on the issue.
You can check here- https://github.com/hackforla/product-management/issues/80

**Comment by DMLROG** [comment]
@ExperimentsInHonesty @SheIsBukki 

Progress: I have been looking through and gathering examples/resources from Hack for LA projects.
I made a more detailed comment about my progress and blockers here.
https://github.com/hackforla/product-management/issues/80#issuecomment-1136951610

Blockers: I find files (that I usually don't have drive access to) and requesting access takes some time. Do you think granting guides team members drive access from the jump is a good idea? @ExperimentsInHonesty 

Availability: 4 hours.

ETA: Can't say for sure.

Pictures (if necessary): None at the moment.

**Comment by SheIsBukki** [comment]
@DMLROG Please list out the files/drives you don't have access to in your guide issue.

**Comment by DMLROG** [comment]
@SheIsBukki @ExperimentsInHonesty 
Yes, those are the files I currently don't have access to. I'll reach out if I find any more like those.
Great. I'll be available for the June 1st meeting then.
And apologies for the lack of consistent updates, I'll give updates more frequently from now on.

**Comment by SheIsBukki** [comment]
@DMLROG Please provide an update

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by DMLROG** [comment]
> @DMLROG
> 
> > Blockers: I find files (that I usually don't have drive access to) and requesting access takes some time. Do you think granting guides team members drive access from the jump is a good idea?
> 
> It would not mirror the experience of the general members of the community of practice who won't have access to all the files across the organization. We keep the files in project silos so that if we do have a data breach it does not affect more than one project.

@ExperimentsInHonesty Yes. I spoke to @SheIsBukki about it at it made sense controlling or securing access to the data so that's okay.

**Comment by DMLROG** [comment]
Progress: Found some more resources and I've gone through the GreenEarthOS project now.
Blockers: Need access to some files. I made a list of all of them in my comment here- https://github.com/hackforla/product-management/issues/80#issuecomment-1158904349
Availability: 3 hours.
ETA: Still can't say yet.
Pictures (if necessary): None at the moment.

**Comment by DMLROG** [comment]
@ExperimentsInHonesty @SheIsBukki 

Progress: —
Blockers: I'm currently moving apartments so I haven't been able to do much. I should have an update by next week when all this is over.
Availability: —
ETA: —
Pictures (if necessary): —

**Comment by SheIsBukki** [comment]
@DMLROG Please provide an update

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

**Comment by SheIsBukki** [comment]
@DMLROG 

We did not hear from you last week after we reached out, please provide the following update in this issue.  
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

If we do not hear from you in the next week, you will be removed from the project, until you are ready to come back.

**Comment by ExperimentsInHonesty** [comment]
@DMLROG We still have not heard from you.  We will remove you from the project on Friday if we don't hear from you by then.

Please provide the following update in this issue.

Progress: "What is the current status of your project? What have you completed and what is left to do?"
Blockers: "Difficulties or errors encountered."
Availability: "How much time will you have this week to work on this issue?"
ETA: "When do you expect this issue to be completed?"
Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."
If we do not hear from you in the next week, you will be removed from the project, until you are ready to come back.

**Comment by DMLROG** [comment]
Progress: Looked through 8 more projects since the last update.
A more detailed read can be found in my comment under the issue.
https://github.com/hackforla/product-management/issues/80#issuecomment-1192493425
Blockers: Files I need access granted to.
Availability: 3 hours.
ETA: Can't say.
Pictures (if necessary): None.

**Comment by SheIsBukki** [comment]
@ExperimentsInHonesty He needs access to these documents https://github.com/hackforla/product-management/issues/80#issuecomment-1192493425

**Comment by DMLROG** [comment]
Progress: Looked through all projects now. Time for a detailed look at FoodOasis and GreenEarthOS.

Blockers: Same as last comment- The files I need access granted to.
Availability: 2 hours.
ETA: Can't say.
Pictures (if necessary): None.

**Comment by DMLROG** [comment]
@ExperimentsInHonesty @SheIsBukki 
https://github.com/hackforla/product-management/issues/80#issuecomment-1287755409

**Comment by edwardsarah** [comment]
Closed due to inactivity. If you'd like to rejoin the guides team, please reach out in the guides slack channel.
